test/unit/aarch64/ArchitectureTest.cc

```diff
@@ -79,13 +79,13 @@ class AArch64ArchitectureTest : public testing::Test {
   })YAML");
 
   // fdivr z1.s, p0/m, z1.s, z0.s
-  std::array<uint8_t, 4> validInstrBytes = {0x01, 0x80, 0x8c, 0x65};
+  const std::array<uint8_t, 4> validInstrBytes = {0x01, 0x80, 0x8c, 0x65};
   std::array<uint8_t, 4> invalidInstrBytes = {0x20, 0x00, 0x02, 0x8c};
```

Comment:
Why is only one of the vectors made const?

Reply:
Unsure will repeat for the second